src/Token/Access.php

```diff
@@ -26,8 +26,8 @@ class Access extends AbstractToken
     public function toHeader(
         $url,
         Config $config,
-        array $customParams = null,
-        $realm = null
+        ?array $customParams = null,
+        ?string $realm = null
```

Comment:
BC Break here too

Reply:
@gsteel Can you please check the PR now? It should be okay now.